core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -329,27 +329,25 @@ final class ShuffleBlockFetcherIterator(
 
   private def createFetchRequest(
       blocks: Seq[FetchBlockInfo],
-      address: BlockManagerId,
-      curRequestSize: Long): FetchRequest = {
-    logDebug(s"Creating fetch request of $curRequestSize at $address "
+      address: BlockManagerId): FetchRequest = {
+    logDebug(s"Creating fetch request of ${blocks.map(_.size).sum} at $address "
```

Comment:
isn't `curRequestSize` also the sum of input blocks size?

Reply:
In non batch mode,  it is. But in batch mode, `blocks` here can a part of all blocks as they will be grouped by `maxBlocksInFlightPerAddress` before calling `createFetchRequest`.